packages/cms-lib/fileManager.js

```diff
@@ -170,6 +183,9 @@ async function getAllPagedFiles(portalId, folderPath) {
  * @param {object} options
  */
 async function fetchFolderContents(portalId, folder, dest, options) {
+  if (isHidden(folder)) {
+    return;
+  }
```

Comment:
Is it possible to handle these when fetching instead of filtering them out here?

Reply:
I had looked at doing that briefly, will take another look

Reply:
I think doing it via the API request might match up well with #248 if we do something to provide the option of fetching hidden files.

Reply:
I switched over to filtering hidden/archived out in the API request. The exception is that `files/stat` does not support filtering, so there I'm still checking manually for singe file fetches.